pypreprocess/conf_parser.py

```diff
@@ -294,12 +294,21 @@ def _ignore_subject(subject_id):
         subject_data_dirs = [x for x in sorted(glob.glob(subject_dir_wildcard))
                              if os.path.isdir(x)]
     else:
+        # list of subjects or subject wildcards
         subject_data_dirs = [os.path.join(dataset_dir, x)
                              for x in subject_data_dirs]
-    sess_func_wildcards = [k for k in options.keys()
-                           if re.match("session_.+_func", k)]
-    sess_onset_wildcards = [k for k in options.keys()
-                            if re.match("session_.+_onset", k)]
+        subject_dir_wildcard = subject_data_dirs
+        aux = []
+        for subject_data_dir in subject_data_dirs:
+            print subject_data_dir
```

Comment:
Oops! I Inserted those print statements for debugging and forgot to remove
them. Thanks for catching.

On Tue, Jan 26, 2016 at 11:33 PM, bthirion <notifications@github.com> wrote:

> In pypreprocess/conf_parser.py
> <https://github.com/neurospin/pypreprocess/pull/183#discussion_r50912156>:
>
> >          subject_data_dirs = [os.path.join(dataset_dir, x)
> >                               for x in subject_data_dirs]
> > -    sess_func_wildcards = [k for k in options.keys()
> > -                           if re.match("session_.+_func", k)]
> > -    sess_onset_wildcards = [k for k in options.keys()
> > -                            if re.match("session_.+_onset", k)]
> > +        subject_dir_wildcard = subject_data_dirs
> > +        aux = []
> > +        for subject_data_dir in subject_data_dirs:
> > +            print subject_data_dir
>
> Do you want to leave such print statements (not Python 3 compatible bzw) ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/neurospin/pypreprocess/pull/183/files#r50912156>.
>



-- 
Elvis Dohmatob,
*sent from my  potato...*
http://dohmatob.blogspot.fr/, https://team.inria.fr/parietal/elvis/,
https://github.com/dohmatob